MidCamp/midcamp
# Video field for Topic nodes

# Description

Satisfies ["@TODO: video embed field (topic)"](https://docs.google.com/document/d/1WBe3lO-6nPO0i5yfXNNujQMLJ4XGEui3Plh4bcAJwDk/edit#).

- Downloads Video Embed Media, Video Embed Field
- Enables Video Embed Media, Video Embed Field, Entity, Media Entity
- Adds a Video media bundle at http://midcamp.org.docker.amazee.io/admin/structure/media/manage/video
- Adds a video embed field to the Topic node

# To Test

- `composer install`
- `drush cim -y`
- Edit a Topic node.  Observe that a video embed field now exists

# Screenshot
![midcamp org docker amazee io_topic_american-medical-association-topic-landing-pages-d8-case-study](https://user-images.githubusercontent.com/4048700/36392659-f393afee-1571-11e8-9c6b-f24465231530.png)




# Notes
I added the Video field just below the body, as my assumption is that it will be for the presentation after MidCamp is complete.  It seemed like a logical place to drop the video.

I hid the field label, because it seemed redundant.
